SPEC.md

```diff
@@ -323,6 +328,20 @@ An example trace:
 
 ```
 
+### Measurements
+
+Measurements are a separate kind of document that engines can emit. Measurements
+are a numeric value bound to a named key. One common use of measurements is
+representing of internal engine metrics. For example, an engine may emit a
```

Comment:
Should we ✂️ this second "of"? -> representing internal engine metrics

Reply:
Woops, thanks.